openassessment/templates/openassessmentblock/oa_submission_answer.html

```diff
@@ -10,7 +10,7 @@ <h2 class="submission__answer__part__prompt__title">{% trans "The question for t
                 </div>
             </article>
             <div class="submission__answer__part__text">
-                <h2 class="submission__answer__part__text__title">{% trans answer_text_label %}</h2>
+                <h2 class="submission__answer__part__text__title">{{ answer_text_label }}</h2>
```

Comment:
rubric has been fixed up - the strings present there did not need to be translated, so the fact that they weren't being translated is okay. I've reverted the `trans` added to those 2 lines in https://github.com/edx/edx-ora2/commit/7d599598f70dc5edefa7ef1e2817a0542c8d198e, see https://github.com/edx/edx-ora2/commit/6bf811b1f8f3d6b07ed7b73dd3a0c72fa5257b6b for details

In `oa_base.html`, it looks as if that variable is being properly translated somewhere along the line. Context: https://github.com/edx/edx-ora2/blob/ora-staff-grading/openassessment/templates/openassessmentblock/oa_base.html#L25
<img width="1220" alt="screen shot 2016-01-07 at 11 26 59 am" src="https://cloud.githubusercontent.com/assets/7373924/12175778/125b03a2-b532-11e5-8da9-edc672126841.png">